parm/config/config.base.emc.dyn

```diff
@@ -184,6 +190,28 @@ case "${APP}" in
     export CCPP_SUITE="FV3_GFS_v17_coupled_p8"
     export confignamevarfornems="cpld${aero_nems_string}"
     ;;
+  NG-GODAS)
+    export REALTIME="NO"
+    export DO_COUPLED="YES"
+    export ATMTYPE="DATA"
+    export DO_POST="NO"
+    export DO_WAVE="NO"
+    export DO_OCN="YES"
+    export DO_ICE="YES"
+    export DO_METP="NO"
+    export DO_VRFY="NO"
+    export RUN="data"
+    export flux_convergence=0.0
```

Comment:
All these model variables should be moved to the `config.defaults.datm`

Reply:
Done.